config/manager/manager.yaml

```diff
@@ -21,6 +21,10 @@ spec:
         args:
         - --leader-elect
         env:
+        - name: LOG_STRUCTURED
```

Comment:
I think you need to run `make manifests` (we need to have these changes in deploy/operator.yaml and in deploy/bundle.yaml too)

Reply:
Ah yes, correct.